thirdeye/thirdeye-frontend/app/pods/components/rootcause-header/template.hbs

```diff
@@ -26,9 +26,16 @@
           key-press="onChange"}}
       {{/unless}}
     </div>
-    <div class="rootcause-header__side">
-      <button {{action "onSave"}} class="te-button">Save</button>
-    </div>
+    {{#if canSave}}
+      <div class="rootcause-header__side">
+        <button {{action "onSave"}} class="te-button">Save</button>
+      </div>
+    {{/if}}
+    {{#if canCopy}}
+      <div class="rootcause-header__side">
```

Comment:
nit: let's make the css class name for meaningful because in general, we shouldn't name things based on their position since that can change. Maybe name it `rootcause-header__button` instead.

Reply:
fixed